mat64/io_test.go

```diff
@@ -39,3 +49,36 @@ func TestDenseRW(t *testing.T) {
 		}
 	}
 }
+
+func TestVectorRW(t *testing.T) {
+	for i, test := range []*Vector{
+		NewVector(0, []float64{}),
```

Comment:
It is really just a question.  I think it would return an EOF error if `n > len(data)`, and panic when `n < 0`.  That sounds good to me, but I wanted it to be considered.